experiments/rollouts_from_policies.sh

```diff
@@ -19,6 +19,7 @@ source experiments/common.env
 
 EXPERT_MODELS_DIR=${EXPERT_MODELS_DIR:-data/expert_models}
 CONFIG_CSV=${CONFIG_CSV:-experiments/rollouts_from_policies_config.csv}
+# TODO(shwang): Set output directory to a temporary directory for testing.
```

Comment:
move comment into "f" conditional

Reply:
"This will prevent tests from generating artifacts that overwrite good expert models "